src/hooks/useGlobalAppState.ts

```diff
@@ -32,6 +36,10 @@ const handleWakeUp = async (
   }
 
   await handleSecurityFlowWakeUp(client)
+
+  // On appStart the homeview will hide the splashscreen
+  // On wakeup, it is more ambiguous so we force it
+  if (!isAppStart) await hideSplashScreen(lockScreens.LOCK_SCREEN)
```

Comment:
nit: If you use `lockScreens.LOCK_SCREEN` enum here, then you should use it also in line 23